.circleci/config.yml

```diff
@@ -40,7 +45,36 @@ jobs:
 
       - run: npm install
       - run: git config --global push.default simple
-      - run: npm run release
+      - run: npm run build:package
+      - run: cd ./build
+      - run: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/repo/.npmrc
+      - run: npm publish --access public
```

Comment:
can we just grab it directly from github? what for do we need npm?

Reply:
with npm we can't unpublish it easily

Reply:
Sure but we can contact npm to take it down if we ever cross that bridge. Worse case we just deprecate it.